.github/pr-labeler.yml

```diff
@@ -0,0 +1,5 @@
+feature: ['feature/*', 'feat/*']
+restyled: ['restyled/*]
+demo: ['demo/*']
```

Comment:
have we ever used that? ^^

Can one for `test` and `chore`?

Reply:
Do you mean "demo"? I am not sure, but why not.. my only motivation was it to label the automatically created restyled PRs, so i thought I also add the other things^^

The requirement that this action bring value is that everyone is creating meaningful branches..

Reply:
We haven't a label for "test". I will add the code, if you add the label^^

Reply:
Just fyi restyled could label it's branches it self :D